Author: Tz-Huan
Subject: Backport upstream commit 964b22216def41e07349cf60db399105b515283a

diff --git a/extras/immodules/client-clutter/scim-bridge-client-imcontext-clutter.c b/extras/immodules/client-clutter/scim-bridge-client-imcontext-clutter.c
index 3b6c46a..784c2b6 100644
--- a/extras/immodules/client-clutter/scim-bridge-client-imcontext-clutter.c
+++ b/extras/immodules/client-clutter/scim-bridge-client-imcontext-clutter.c
@@ -26,6 +26,7 @@
 #include <X11/Xlib.h>
 #include <clutter/clutter.h>
 #include <clutter/x11/clutter-x11.h>
+#include <clutter-imcontext/clutter-imcontextsimple.h>
 
 #include "scim-bridge-attribute.h"
 #include "scim-bridge-client.h"

